backend/hitas/templates/components/improvements_table.jinja

```diff
@@ -147,6 +161,15 @@
                 <td class="improvement-cell text-right">{% if improvement.depreciation is not none %}{{ improvement.depreciation.amount|intcomma }}{% else %}-{% endif %}</td>
                 <td class="improvement-cell text-right">{{ improvement.accepted_value_for_housing_company|intcomma }}</td>
                 <td class="improvement-cell text-right">{{ improvement.accepted_value|intcomma }}</td>
+                {% if excess and excess.value_per_square_meter_before_2010 is not none and excess.value_per_square_meter_after_2010 is not none %}
+                <td class="improvement-cell text-right">
+                  {% if improvement.completion_date|date_before("2010-01-01") or excess.value_per_square_meter_before_2010 is none %}
+                    <small>1</small>
+                  {% else %}
+                    <small>2</small>
+                  {% endif %}
```

Comment:
Nice and simple way to check for the date, instead of saving the excess for every improvement 👍